framework/src/modules/chain/submodules/loader.js

```diff
@@ -867,7 +867,7 @@ __private.loadBlocksFromNetwork = function(cb) {
  */
 __private.sync = function(cb) {
 	library.logger.info('Starting sync');
-	if (components.cache.cacheReady) {
+	if (components.cache.options.enabled && components.cache.cacheReady) {
```

Comment:
I would remove the second condition as it isn't necessary and creates confusion

```suggestion
	if (components.cache.options.enabled) {
```

Reply:
Cache can be enabled and not ready (for example redis was not up at boot or went down)